gremlin-scala/src/main/scala/gremlin/scala/GremlinScala.scala

```diff
@@ -364,6 +364,18 @@ case class GremlinScala[End, Labels <: HList](traversal: GraphTraversal[_, End])
     GremlinScala[A, Labels](traversal.union(mappedTraversals: _*))
   }
 
+  def choose[A, B](
+    predicate: GremlinScala[End, HNil] ⇒ GremlinScala[A, _],
+    trueChoice: GremlinScala[End, HNil] ⇒ GremlinScala[B, _],
```

Comment:
I just used what was in the the TinkerPop Java. I like `onTrue`, etc. better too. Shall I change it?